inz-nsn/description.xml

```diff
@@ -1,11 +1,11 @@
 <Issuer version="4">
-    <ID>inz-nhi</ID>
```

Comment:
At the moment it is not possible to change the `ID` of existing credentials. IRMA apps don't support that. You can either only change the name and keep the existing identifier, or you can make a new credential with the new `ID`.